src/ast/passes/codegen_llvm.cpp

```diff
@@ -1706,7 +1721,13 @@ void CodegenLLVM::binop_int(Binop &binop)
     default:
       LOG(BUG) << "\"" << opstr(binop) << "\" was handled earlier";
   }
-  // Using signed extension will result in -1 which will likely confuse users
+
+  return;
+
+sign_extend_bool:
```

Comment:
Does it have to be a goto? Why not just do this in place of the goto?
`expr_ = b_.CreateIntCast(expr_, b_.getInt64Ty(), false);`

Reply:
Ack. Will just inline it.